docs/tracking-tool-for-climate-watch/components.md

```diff
@@ -235,51 +276,25 @@ None
 - [lodash/debounce](https://www.npmjs.com/package/lodash.debounce)
 - [classnames](https://www.npmjs.com/package/classnames)
 
-## Dropdown
-
-> ![](../assets/components/dropdown.png)
-
-> ![](../assets/components/dropdown-open.png)
+## Nav
 
-> ![](../assets/components/dropdown-disabled.png)
+> ![](/.gitbook/assets/components/navbar.png)
```

Comment:
Please use local paths as they wont show in the production gitbook otherwise (/.gitbook/assets -> ../assets)